Makefile

```diff
@@ -5,6 +5,16 @@ PRODUCTION_URL="https://docs.mongodb.com"
 STAGING_BUCKET=docs-mongodb-org-stg
 PRODUCTION_BUCKET=docs-mongodb-org-prd
 
+ifeq ($(ENV), 'dotcom_stg')
+	STAGING_URL="https://docs-mongodborg-staging.corp.mongodb.com"
+	STAGING_BUCKET=docs-mongodb-org-dotcomstg
+endif
+
+ifeq ($(ENV), 'dotcom_prd')
```

Comment:
Writers will want to stage and publish to the same environment. Let's have one variable check for "dotcom" and set both the staging and prod URLs/buckets in one shot.